presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGenerator.java

```diff
@@ -374,4 +377,26 @@ public void testAggregationWithOrderByPushDownInTopN()
         TopNNode topN = new TopNNode(planBuilder.getIdAllocator().getNextId(), agg, 50L, new OrderingScheme(ImmutableList.of(new Ordering(variable("city"), SortOrder.DESC_NULLS_FIRST))), TopNNode.Step.FINAL);
         testPinotQuery(pinotConfig, topN, "", defaultSessionHolder, ImmutableMap.of());
     }
+
+    @Test
```

Comment:
Again pardon my ignorance on the Pinot-SQL PR: But wasn't there a special derived test class to test the PQL one alone. This looks to be testing the PQL flow -- should it go in its own class ?

Reply:
you are right, will merge the tests.

Reply:
done